assets/airports/ulli.json

```diff
@@ -0,0 +1,2695 @@
+            ]
+        },
+        "TURAL1C": {
+            "icao": "TURAL1C",
+            "name": "Tural One Charlie",
+            "rwy": {
+                "ULLI10R": ["LI784", ["LI767", "S225-"], ["LI702", "S225-"], ["LI703", "S240-"], "LI704", ["LI705", "A110+"], ["TURAL", "A120+"]]
+            },
+            "exitPoints": {
+                "TURAL": []
+            },
+            "draw": [
+                ["LI784", "LI767", "LI702", "LI703", "LI704", "LI705", "TURAL*"]
+            ]
+        },
+        "TURAL1D": {
+            "icao": "TURAL1D",
+            "name": "Tural One Delta",
+            "rwy": {
+                "ULLI10L": ["LI783", ["LI701", "S225-"], ["LI702", "S225-"], ["LI703", "S240-"], "LI704", ["LI705", "A110+"], ["TURAL", "A120+"]]
+            },
+            "exitPoints": {
+                "TURAL": []
+            },
+            "draw": [
+                ["LI783", "LI701", "LI702", "LI703", "LI704", "LI705", "TURAL*"]
+            ]
+        },
+        "UTETA1A": {
+            "icao": "UTETA1A",
+            "name": "Uteta One Alpha",
+            "rwy": {
+                "ULLI28R": ["LI719", ["LI709", "S225-"], ["UTETA", "A80+"]]
+            },
+            "exitPoints": {
+                "UTETA": []
+            },
+            "draw": [
+                ["LI719", "LI709", "UTETA*"]
+            ]
+        },
+        "UTETA1B": {
+            "icao": "UTETA1B",
+            "name": "Uteta One Bravo",
+            "rwy": {
+                "ULLI28L": ["LI785", ["LI768", "S225-"], ["UTETA", "A80+"]]
+            },
+            "exitPoints": {
+                "UTETA": []
+            },
+            "draw": [
+                ["LI719", "LI709", "UTETA*"]
+            ]
+        },
+        "UTETA1C": {
+            "icao": "UTETA1C",
+            "name": "Uteta One Charlie",
+            "rwy": {
+                "ULLI10R": ["LI784", ["LI767", "S225-"], ["LI702", "S225-"], ["LI703", "S240-"], "LI704", ["LI706", "A110+"], ["UTETA", "A120+"]]
+            },
+            "exitPoints": {
+                "UTETA": []
+            },
+            "draw": [
+                ["LI784", "LI767", "LI702", "LI703", "LI704", "LI706", "UTETA*"]
+            ]
+        },
+        "UTETA1D": {
+            "icao": "UTETA1D",
+            "name": "Uteta One Delta",
+            "rwy": {
+                "ULLI10L": ["LI783", ["LI701", "S225-"], ["LI702", "S225-"], ["LI703", "S240-"], "LI704", ["LI706", "A110+"], ["UTETA", "A120+"]]
+            },
+            "exitPoints": {
+                "UTETA": []
+            },
+            "draw": [
+                ["LI783", "LI701", "LI702", "LI703", "LI704", "LI705", "UTETA*"]
+            ]
+        }
+    },
+    "stars": {
+        "AKARU1A": {
+            "icao": "AKARU1A",
+            "name": "Akaru One Alpha",
+            "entryPoints": {
+                "AKARU": [["AKARU", "A150-"]]
+            },
+            "body": [
```

Comment:
Although this was admittedly easier to read, body sections should be all in one line, for consistency's sake :sweat_smile: